packages/react/src/components/HorizontalRule/__stories__/HorizontalRule.stories.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, select, text } from '@storybook/addon-knobs';
+import '@ibmdotcom/styles/scss/components/horizontalrule/_horizontalrule.scss';
+
+import HorizontalRule from '../HorizontalRule';
+
+storiesOf('HorizontalRule', module)
+  .addDecorator(withKnobs)
+  .add('Default', () => {
+    const styles = {
+      solid: 'solid',
```

Comment:
I'm relatively new to Storybook myself, but would setting something like this to `false` prevent it from adding unnecessary class names to the component?